app/src/main/java/com/example/makeiteven2/MainActivity.kt

```diff
@@ -61,7 +61,7 @@ class MainActivity : AppCompatActivity(), FragmentStartScreen.IFragmentsStartsSc
 
         mSharedPref = applicationContext.getSharedPreferences(Constants.SHARED_PREFS, Context.MODE_PRIVATE)
         mEditor = mSharedPref.edit()
-        mAudioManager = AudioManager(this).getInstance(this)
+        mAudioManager = AudioManager(this).getInstance(this) // mAudioManager from constants
```

Comment:
remove constructor 
do inside AudioManager surround with companion object to make it"static"